source/compiler/sc2.c

```diff
@@ -1231,6 +1231,31 @@ static int command(void)
           cell val;
           preproc_expr(&val,NULL);
           sc_needsemicolon=(int)val;
+        } else if (strcmp(str,"once")==0) {
+          char symname[sNAMEMAX];
+          symbol *included;
+          /* make it compatible for Windows paths with '\\' directory separator */
+          char* basefname=inpfname;
+          for (char* _cchar=inpfname; *_cchar!='\0'; _cchar++)
+            if (*_cchar=='/'||*_cchar=='\\')
+              basefname=_cchar+1;
+          /* assign '_inc_includename' string to symname */
+          strcpy(symname,"_inc_");
+          strlcat(symname,basefname,sizeof symname);
```

Comment:
How closely does this replicate all the behaviour here: https://github.com/pawn-lang/compiler/wiki/File-Inclusion